.github/workflows/buildAndPublishReactNativeArtifacts.yml

```diff
@@ -0,0 +1,213 @@
+            exit 1
+          fi
+
+          PREVIOUS_PATCHES_HASH=$(./.github/scripts/computePatchesHash.sh)
+
+          if [ "$CURRENT_PATCHES_HASH" == "$PREVIOUS_PATCHES_HASH" ]; then
+            echo "No changes in HybridApp patches"
+            exit 1
+          fi
+
+          echo "Detected changes in HybridApp patches:"
+          git -C Mobile-Expensify diff --name-only $PREVIOUS_SUBMODULE_COMMIT..$CURRENT_SUBMODULE_COMMIT -- "patches/react-native+*.patch"
+        env:
+          PATCHES_PATHS: "Mobile-Expensify/patches"
+          VERSION: ${{ env.VERSION }}
+
+  buildAndPublishRNArtifactsForStandaloneNewDot:
+    runs-on: ubuntu-latest
+    if: ${{ always() && needs.verifyNewDotPatchesAndReactNativeVersion.result == 'success'}}
+    needs: verifyNewDotPatchesAndReactNativeVersion
+    steps:
+      - name: Checkout
+        # v4
+        uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608
+
+      - name: Get react-native version
+        run: |
+          echo "VERSION=$(jq -r '.dependencies["react-native"]' package.json)" >> $GITHUB_ENV
+
+      - name: Setup Node
+        uses: ./.github/actions/composite/setupNode
+
+      - name: Setup Java
+        # v4
+        uses: actions/setup-java@3a4f6e1af504cf6a31855fa899c6aa5355ba6c12
+        with:
+          distribution: oracle
+          java-version: 17
+
+      - name: Setup Gradle
+        # v4
+        uses: gradle/actions/setup-gradle@06832c7b30a0129d7fb559bcc6e43d26f6374244
+
+      - name: Save patches hash
+        run: |
+          echo "PATCHES_HASH=$(./.github/scripts/computePatchesHash.sh)" >> $GITHUB_ENV
+        env:
+          VERSION: ${{ env.VERSION }}
+          PATCHES_PATHS: "patches"
+
+      - name: Determine new patched RN version
+        run: |
+          echo "NEW_PATCHED_VERSION=$(./.github/scripts/getNewPatchedRNVersion.sh)" >> $GITHUB_ENV
+        env:
+          VERSION: ${{ env.VERSION }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          IS_HYBRID_BUILD: "false"
+
+      - name: Build and publish RN artifacts
+        run: |
+          cd android
+          echo "Starting artifacts build for standalone NewDot"
+          echo "Version: ${{ env.NEW_PATCHED_VERSION }}"
+          echo "Commit hash: ${{ env.COMMIT_HASH }}"
+          echo "Patches hash: ${{ env.PATCHES_HASH }}"
+          export ORG_GRADLE_PROJECT_reactNativeArchitectures="armeabi-v7a,arm64-v8a,x86,x86_64"
+          ./gradlew buildReactNativeArtifacts -x lint -x test -x check
+          ./gradlew publishReactNativeArtifacts
+        env:
+          GH_PUBLISH_ACTOR: ${{ github.actor }}
+          GH_PUBLISH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          IS_HYBRID_BUILD: "false"
+          PATCHED_VERSION: ${{ env.NEW_PATCHED_VERSION }}
+          COMMIT_HASH: ${{ github.event.after }}
+          PATCHES_HASH: ${{ env.PATCHES_HASH }}
+
+  buildAndPublishRNArtifactsForHybridApp:
```

Comment:
This is almost the same as `buildAndPublishRNArtifactsForStandaloneNewDot`. Maybe I’ll manage to create a composite action.